modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -117,75 +140,180 @@ template<typename Net> class Params {
               , 1u} {
     };
 
-    Params<Net>& cfgInputLayers(const typename PortCfg<Net>::In &ll) {
+    /** @brief Specifies sequence of network input layers names for inference.
+
+    The function is used to associate data of graph inputs with input layers of
+    network topology. Number of names has to match the number of network inputs. If a network
+    has only one input layer, there is no need to call it as the layer is
+    associated with input automatically but this doesn't prevent you from
+    doing it yourself. Count of names has to match to number of network inputs.
```

Comment:
avoid constructions like this, avoid "you" and "yourself" in the documentation too.

Reply:
Done